### DISCUSSION TITLE
BorgBackup-install

### DISCUSSION THREAD

**Discussion by jedie** [borgbackup/borg#5621] [Show and tell]
I made https://github.com/jedie/borgbackup-install for easy install/update Borg via pip in a venv.

Any comments?

**Comment by ThomasWaldmann** [comment]
For borg 1.1.x and recent debian/ubuntu: zstd, lz4, b2 and xxhash is missing, see:

https://github.com/borgbackup/borg/blob/1.1-maint/docs/installation.rst#dependencies

(and I just noticed that xxhash is not documented yet there, it was recently made possible to have that externally)

borg has that stuff bundled, but for better security maintenance, it is preferable to use the distribution packages IF the dist provides a suitable package. if the respective -dev packages are installed, borg setup should find the stuff via pkg-config.

**Comment by ThomasWaldmann** [comment]
https://github.com/borgbackup/borg/blob/1.1-maint/docs/installation.rst#debian--ubuntu there is also some update needed along what I posted above.
